src/lib/device-types/index.ts

```diff
@@ -49,6 +50,14 @@ const SPECIAL_SLUGS = ['edge'];
 const RETRY_DELAY = 2000; // ms
 const DEVICE_TYPES_CACHE_EXPIRATION = 5 * 60 * 1000; // 5 mins
 
+const syncSettings: { map: Dictionary<string> } = {
+	map: {},
+};
+
+export function setSyncMap(map: typeof syncSettings['map']) {
```

Comment:
This function is unused. If you plan to use it as an extension point from resin-api, then please add a comment to that effect.

Reply:
Good catch. I forgot to call it in the onInitHook function.